cmd/crc/cmd/config/config_darwin.go

```diff
@@ -6,6 +6,8 @@ import (
 
 var (
 	// Preflight checks
+	SkipCheckVirtualBoxInstalled     = cfg.AddSetting("skip-check-virtualbox-installed", nil, []cfg.ValidationFnType{cfg.ValidateBool}, []cfg.SetFn{cfg.SuccessfullyApplied})
+	WarnCheckVirtualBoxInstalled     = cfg.AddSetting("warn-check-virtualbox-installed", nil, []cfg.ValidationFnType{cfg.ValidateBool}, []cfg.SetFn{cfg.SuccessfullyApplied})
```

Comment:
@gbraad @anjannath is it true that we are completely not supporting VB on windows. iirc if a user can setup his dns server then he can still use VB since cluster should be working internally?

Reply:
I haven't tried VB in windows, but i remember @gbraad mentioning there were some issues with the networking interface. So yes atm it is not supported!

Reply:
Removing these options from the windows build does not mean we are not supporting vbox on Windows. It only acknowledges the fact that we have no preflight checks/fixes for vbox on windows, so these config flags are not doing anything at the moment. If/when we add vbox preflight checks on Windows, then we'll readd these config flags.

Reply:
the issue for vbox is worked around for `dns_windows.go` when vbox is used